CHANGELOG.md

```diff
@@ -4,8 +4,11 @@
 
 ### Breaking changes
 
+- The `<GooglePayButton />` component no longer automatically uses the dark version when the device is in Dark Mode. Instead, you should detect Dark Mode in your app and use the `<GooglePayButton />` `type` prop appropriately (according to [Google's guidelines](https://developers.google.com/pay/api/android/guides/brand-guidelines)). [#1051](https://github.com/stripe/stripe-react-native/pull/1051)
```

Comment:
Could the library still support the native light/dark mode? Something like, you can force it to be light, dark, or not set any value and it will match the current theme?

Reply:
It looks like that's what it does in `GooglePayButtonView.kt` if you don't set a type, or set it to an invalid value, but this descriptions says otherwise.

Reply:
Ah yeah- it _does_ do exactly what you said right now: you can force it to be light, dark, or not set any value and it will match the current theme

The change is that previously, even if you set it to a light button type explicitly, if the device was in dark mode, it would override that and set the dark button. Honestly this isn't much of a breaking change...more of a bug fix 😅  wdyt?

Reply:
I say move this to bug fix and say 

```suggestion
- The `<GooglePayButton />` component no longer overrides the `type` when the device is in Dark Mode and forces a dark button. Instead, you should detect Dark Mode in your app and use the `<GooglePayButton />` `type` prop appropriately (according to [Google's guidelines](https://developers.google.com/pay/api/android/guides/brand-guidelines)). [#1051](https://github.com/stripe/stripe-react-native/pull/1051)
```

Reply:
I agree with the way it's implemented then, but the comment here is still confusing, especially the last part where you're still telling people to detect Dark Mode themselves. What about this:

- The `<GooglePayButton />` component no longer overrides the `type` to use the dark mode version when the device is in Dark Mode. If you set the `type` value, it will always be respected. If you don't set the `type` value, it will match system's theme. [#1051](https://github.com/stripe/stripe-react-native/pull/1051)

Reply:
im good with that, just a small addition at the end:
```suggestion
- The `<GooglePayButton />` component no longer overrides the `type` to use the dark mode version when the device is in Dark Mode. If you set the `type` value, it will always be respected. If you don't set the `type` value, it will match system's theme (`standard_shadow` when in Light Mode, and `standard_dark` when in Dark Mode). [#1051](https://github.com/stripe/stripe-react-native/pull/1051)
```